integration_test/manifests/connection-manager-api/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: connection-manager-api
+  labels:
+    app.kubernetes.io/name: connection-manager-api
+spec:
+  type: "NodePort"
+  ports:
+  - name: "3001"
```

Comment:
I don't _think_ the name is mandatory (don't quote me..), but in any case, I'd think probably this should be either more descriptive, or just omitted.

Reply:
Will do. I think I copy/pasted this from somewhere...